src/core.js

```diff
@@ -2007,6 +2008,8 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
       datamap.destroy();
     }
 
+    data = instance.runHooks('beforeLoadData', data, firstRun, source);
```

Comment:
Can I ask to update TS definition file? There is a missing `source` argument for both related hooks.  
 * [beforeLoadData](https://github.com/handsontable/handsontable/blob/f1dd40a2b008f765736454ded538bcbd01149dfb/handsontable.d.ts#L1879)
 * [afterLoadData](https://github.com/handsontable/handsontable/blob/f1dd40a2b008f765736454ded538bcbd01149dfb/handsontable.d.ts#L1797)
And tests to update [test/types/settings.types.ts](https://github.com/handsontable/handsontable/blob/feature/hyperformula-2021/test/types/settings.types.ts)

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/df936e2bcb2893f2595672f7f90c6cb48aadfe47

Reply:
whoops, correction https://github.com/handsontable/handsontable/pull/7648/commits/608021bc45388112c6bca42a098e52a534f9fb52